### ISSUE TITLE
Blobs recovered from erasure should not be stored in blocktree

### ISSUE THREAD

**Issue by carllin** [solana-labs/solana#5614]
#### Problem
Blobs recovered from erasure don't have the originals leader's signature, so they cannot be propagated to the rest of the network. They are also no longer needed for recovery once recovery is finished. Thus there's no point in storing them.

**Comment by pgarg66** [comment]
This is how shreds are working. Recovered coding shreds are not being stored in blocktree.

**Comment by pgarg66** [comment]
BTW, recovered data shreds have original leader's signature, and can be used for repairs. These are being stored in the blocktree.
